.vscode/extensions.json

```diff
@@ -4,7 +4,7 @@
     "EditorConfig.EditorConfig",
     "syler.sass-indented",
     "firsttris.vscode-jest-runner",
-    "nrwl.angular-console"
+    "nrwl.angular-console",
```

Comment:
This isn't technically valid JSON unless VSCode uses a slightly different variant?

Reply:
You're right did not notice it when I moved prettier back to unwantedRecommendations.